src/MRA.Jobs.Api/Application/Features/Applications/Query/GetApplicationWithPagination/GetApplicationsPagedQueryHandler.cs

```diff
@@ -19,8 +19,9 @@ public Task<PagedList<ApplicationListDto>> Handle(PagedListQuery<ApplicationList
             .Include(a => a.Vacancy)
             .Include(a => a.VacancyResponses)
             .ThenInclude(vr => vr.VacancyQuestion);
-
-        var applications = currentUser.GetRoles().Any(a => a == "Applicant")
+        
+        var roles = currentUser.GetRoles();
```

Comment:
Return all applications when it is reviewer

Reply:
I made it so that the admin could also see applications : var applications = roles.Any()
            ? allApplications
            : allApplications.Where(a =>
                a.ApplicantUsername == currentUser.GetUserName() && a.Vacancy.Discriminator != "NoVacancy");